go.mod

```diff
@@ -1,19 +1,27 @@
 module github.com/conductorone/conductorone-sdk-go
 
-go 1.14
+go 1.20
```

Comment:
Minor nitpick: We were using go v1.21 before. Is that not available in github actions yet?

Reply:
Oh it should be, I will use that version